README.md

```diff
@@ -44,15 +44,15 @@ We welcome you to add support for your favorite database!
     <dependency>
       <groupId>io.vlingo</groupId>
       <artifactId>vlingo-symbio</artifactId>
-      <version>0.2.7</version>
+      <version>0.2.9</version>
```

Comment:
I have already used 0.2.9. Your version should be 0.3.0 or later depending on the latest push to master. Please check when you are ready for merge.